automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestParameterSetting.scala

```diff
@@ -206,12 +203,155 @@ class TestParameterSetting  extends ExtractionTest {
   val t1c = "We therefore assume that S(0) = 6.8 – 0.5 = 6.3 million."
   failingTest should s"extract the parameter setting(s) from t1c: ${t1c}" taggedAs(Somebody) in {
     val desired = Seq(
-      "S(0)" -> Seq("6.3") // fixme: is million a param setting or unit?
+      "S(0)" -> Seq("6.3 million")
     )
     val mentions = extractMentions(t1c)
     testParameterSettingEvent(mentions, desired)
   }
 
+  // SuperMaaS tests
+
+  val u1a = "Default nitrogen removal is 15 % for sheep & beef, and 25 % for dairy grazed pastures but it remains the user's choice to set a sensible value for their system ."
+  failingTest should s"extract the parameter setting(s) from u1a: ${u1a}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "nitrogen removal" -> Seq("15"), //fixme: handle percentages
+      "nitrogen removal" -> Seq("25")
+    )
+    val mentions = extractMentions(u1a)
+    testParameterSettingEvent(mentions, desired)
+  }
+
+  val u2a = "It shows that the pasture is not harvested before 1/07/1995 , the harvest frequency is once every 21 days and the harvest residual is 1250 kg / ha ."
+  failingTest should s"extract the parameter setting(s) from u2a: ${u2a}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "harvest residual" -> Seq("1250") //todo: add this as unit test, too
```

Comment:
Did you want to add this as unit test within this PR? If you did, don't forget to add the rule before merging!

Reply:
added! thanks!